src/scripts/actions/VideoActions.js

```diff
@@ -11,7 +11,7 @@ export const videoDataLoaded = createAction(VIDEO_DATA_LOADED)
 
 export const fetchVideo = (id: string) => async (dispatch: Dispatch<*>) => {
   console.log('fetching video info for video with id: ' + id)
-  let videoInfo = await paratii.eth.vids.get(id)
+  let videoInfo = await paratii().eth.vids.get(id)
```

Comment:
I am proposing this to be the new model for accessing the paratii-lib instance. Simply call `paratii()` to get the singleton, and only after the surrounding app has called `initParatiiLib` asynchronously